tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -154,6 +154,22 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                   try {
                       jobs = [:]
                       println "Branch: ${branch}"
+                      if (rancher_version.startsWith("v2.5")){
+                        if (!env.RKE_VERSION) {
```

Comment:
Does this mean, the following stmt will get executed only if there is no environment variable RKE_VERSION ? But there will be a default RKE_VERSION variable present in the Jenkins always. What about that usecase?

Reply:
This is the `parent` jenkinsfile, which will set the env. var for any child jobs/jenkinsfiles it will run. So This will override the child default versions for these variables.